client/src/app/infrastructure/models/poll/has-poll-ids.ts

```diff
@@ -0,0 +1,20 @@
+import { Id } from 'src/app/domain/definitions/key-types';
+import { ViewPoll } from 'src/app/site/pages/meetings/pages/polls';
+
+export interface HasPolls extends HasPollIds {
+    polls: ViewPoll[];
+}
+
+export interface HasPollIds {
+    poll_ids: Id[]; // (poll/content_object_id)[];
+}
+
+export function isPartialHasPolls(item: any): item is Partial<HasPolls> {
+    let isViewPollArray = true;
+    if (!Array.isArray(item?.polls)) {
+        isViewPollArray = false;
+    } else {
+        isViewPollArray = item.polls.some(value => !(value instanceof ViewPoll));
+    }
+    return typeof item === `object` && (isViewPollArray || typeof item?.poll_ids === `number`);
```

Comment:
Does this work? Since `poll_ids` is an array, its prototype is `object` not `number`